content/en/seachat/seachat-manual/02-create-agent/04-agent-information.md

```diff
@@ -0,0 +1,114 @@
+
+---
+
+## Basic Setting in Agent Information
+
+SeaChat allows you to configure your AI agent's information such as names, use case, and description. Inside **Basic Setting**, users can set the following parameters:
+
+### Name
+The name of your AI agent. This name will be displayed in the conversation window.
+
+> :page_facing_up: **NOTE**
+> 
+> This will be the name of your agent for the purpose og agent management. If you wish to change the displayed name in your agent integration. Please refer to [Webpage Integration](/seachat/seachat-manual/04-channels/08-install-to-webpage/).
+
+
+### Use Case
+The use case of your AI agent. This will help you to categorize your AI agent. SeaChat provides a list of use cases to choose from. Depending on your needs, you can choose a use case that best fits your AI agent. Different use cases might require different description settings, but you can simply follow the instructions in the description section when incorporating a use case.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/use-case-examples.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/use-case-examples.png" alt="image that display the use case options available in SeaChat">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Choose a use case for your AI agent</p>
+</div>
+
+### Description
+A brief description of your AI agent that details your agent's behavior and its reference. Whatever you put inside the description field will show up in agent instructions on the right-hand side of the page. This description will then be used to prompt a language model to generate responses.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/description-preview.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/description-preview.png" alt="image showcasing how to write an agent description">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Agent Description</p>
+</div>
+
+### Response Language
+The language the AI agent will use to respond to the user. If the language that the agent will use is not definitive, you can choose **Always match user input language**. 
```

Comment:
change "not definitive" to "if you want the AI agent to be multilingual and respond based on customer query language, you can choose **Always match user input language**.